README.md

```diff
@@ -1,41 +1,44 @@
-# risc0-rust-starter
+# RISC Zero Rust Starter
 
-This is a small example designed to be a minimum starting project for the RISC Zero zero knowledge virtual machine.
-In this example, we use zero knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are.
+The `risc0-rust-starter` repo is a minimal starting project for zero-knowledge software development. In this RISC Zero "Hello World" project, we use zero-knowledge proof techniques to prove that we know the factors of some composite number `N`, without revealing what the factors are. 
 
-For the main project, see [here](https://github.com/risc0/risc0)
+For more information, check out the [main risc0 repo](www.github.com/risc0/risc0) and the [RISC Zero website](http://www.RISCZero.com).
 
-# Building & running
+# Quick Start
 
-Make sure you are using the nightly version of rust:
+Make sure you have [Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) installed and are using the [nightly](https://doc.rust-lang.org/book/appendix-07-nightly-rust.html) version of [Rust](https://doc.rust-lang.org/book/ch01-01-installation.html):
 
 ```
 rustup toolchain install nightly
 rustup override set nightly
 ```
+To build all methods and execute a multiplication method within the zkVM, run either of the following options: 
```

Comment:
```suggestion
To build all methods and execute the method within the zkVM, run either of the following options: 
```

Reply:
This feels confusing to me but I'm fine with committing this suggestion if it feels clear to folks who are more fluent in software.

I think it would be prudent to add a pointer to a definition of the word method as we're using it. Is the definition we offer on the website appropriate for the way we use the term here? 
https://www.risczero.com/docs/key-terminology

Reply:
I guess my thinking was related to how the intent of this starter project is for developers to be able to clone this project and then make changes for their own app. So I wanted to try and generalize, but maybe that is too confusing.